scripts/README.md

```diff
@@ -0,0 +1,37 @@
+Script to import `at` macro and add it to re-com components.
```

Comment:
I would suggest these words ... 

A script that recursively traverses all the ClojureScript code in an existing codebase, adding `:src  (at)` to every use of a re-com component.   To those namespaces which need updating (ie. those using `re-com` components), it will add the necessary namespace `requires` for `at`.

You should run this script on your existing codebase to take advantage of `re-com's` new `:src` debugging feature described here:
See:  https://re-com.day8.com.au/#/debug

This script is clever enough to detect when a component already has an existing `:src  (at)` argument, and to not add duplicates.  As a result, it can be run multiple times on a codebase.